vocabulary-draft/continuous-delivery.md

```diff
@@ -0,0 +1,114 @@
+# Continuous Delivery
+
+Continuous Delivery related to activities and orchestration that needs to exist to be able to deterministically and continously being able to delivery software to users. 
+
+## Pipeline run events
```

Comment:
I think these kind of events belong to a different level of abstraction, perhaps an orchestration bucket?

Reply:
I think in the last meeting we decided that Continuous Delivery and Orchestration belong in the same bucket, but I can't find a clear statement on it so I might have misunderstood. Personally I would be fine with moving this out to an orchestration bucket.

Reply:
If there would be an orchestration bucket the I agree that the pipeline and task events should be placed there. Without such a bucket I believe that the Cotinuous Delivery bucket could contain them, just as it could contain any other of our events that don't naturally belong to any of the other defined buckets. If I interpret [this proposal for definitions of CI/CD terms](https://github.com/cdfoundation/glossary/pull/10/files?short_path=fef0ae7#diff-fef0ae7d1dfc25c4520dacd30b238721e19074ced4caebc190846f961453b860) correctly I'd say that Continuous Delivery is everything that is not Continuous Integration or Continuous Deployment.